pkg/scheduler/framework/plugins/defaultpreemption/default_preemption_test.go

```diff
@@ -1676,8 +1676,8 @@ func TestPreempt(t *testing.T) {
 				podInformer.GetStore().Add(test.pods[i])
 			}
 
-			deletedPodNames := make(sets.String)
-			patchedPodNames := make(sets.String)
+			deletedPodNames := sets.New[string]()
```

Comment:
It's better to replace all `sets.String{}` in the scheduling part together in one PR.

Reply:
Sure! I will replace all together.

Reply:
+1 for make it in one PR for each components.

Reply:
@denkensk @chendave 
I updated scheduler apis first. Please take a look again.

Reply:
We already have one PR here https://github.com/kubernetes/kubernetes/pull/113805, you can work on the exist one with the agreement of @Bryce-Soghigian if he doesn't have time.

Reply:
Sure I will close this PR